### PR TITLE
Support function calls

### PR DESCRIPTION
**Pull request by umanghome**
Before:
image

After:
image

This will break if the function call spans multiple lines as TextMate Grammar can only match single lines. https://github.com/microsoft/vscode-textmate/issues/32

I'm not very familiar with TextMate Grammars. This might not even be the correct way to solve the problem of function calls.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+12
```
Comment (jasonwilliams) [inline .nvmrc:1]: @umanghome whats this? I don’t think this should be here

**Review by jasonwilliams — CHANGES_REQUESTED** [review]

File: .nvmrc (hunk 1 above)
Comment (jasonwilliams) [reply .nvmrc:1]: It guess it seems useful to people using that

**Review by jasonwilliams — APPROVED** [review]

**Comment by jasonwilliams** [comment]
Thanks for the contribution @umanghome hopefully you can help on other issues too!

**Comment by jasonwilliams** [comment]
@umanghome was there an issue this was tied to?

**Comment by umanghome** [comment]
Not for function calls. I was trying to fix #233 and function calls together.

**Comment by jasonwilliams** [comment]
@umanghome are you still working on #233 then?

**Comment by umanghome** [comment]
#233 can't be fixed as function declarations usually span multiple lines and TextMate Grammar doesn't support matching multiple lines. https://github.com/microsoft/vscode-textmate/issues/32

I'll comment this on the issue and the one linked to it.

**Pull request merged**